controllers/managedocs_controller.go

```diff
@@ -68,6 +69,11 @@ const (
 	managedOCSName                         = "managedocs"
 	storageClusterName                     = "ocs-storagecluster"
 	prometheusName                         = "managed-ocs-prometheus"
+	prometheusServiceName                  = "managed-ocs-prometheus-service"
+	servingCertSecretBetaAnnotationKey     = "service.beta.openshift.io/serving-cert-secret-name"
+	servingCertSecretAlphaAnnotationKey    = "service.alpha.openshift.io/serving-cert-secret-name"
+	servingCertConfigMapBetaAnnotationKey  = "service.beta.openshift.io/inject-cabundle"
+	servingCertConfigMapAlphaAnnotationKey = "service.alpha.openshift.io/inject-cabundle"
```

Comment:
- same as above, mostly the services providing this functionality were promoted to beta so beta would suffice, let me know otherwise

Reply:
same answer as above